dropout1d torch frontend #10904

Nasser-Mohammed opened the pull request:
close #10904

suvadityamuk commented:
Hi, @Nasser-Mohammed!

Thank you for your contribution, much appreciated! :grin: 

It seems that you have not added a test for the function you have written. Do you mind possibly going back to add that? More information on that should be available [here](https://lets-unify.ai/ivy/deep_dive/ivy_frontends_tests.html)! Also, make sure that you use the [pre-commit hook](https://lets-unify.ai/ivy/deep_dive/formatting.html#pre-commit-hook) as well to make sure your commits are linted properly.

Thanks again! :smile:

Nasser-Mohammed commented:
> Hi, @Nasser-Mohammed!
> 
> Thank you for your contribution, much appreciated! 😁
> 
> It seems that you have not added a test for the function you have written. Do you mind possibly going back to add that? More information on that should be available [here](https://lets-unify.ai/ivy/deep_dive/ivy_frontends_tests.html)! Also, make sure that you use the [pre-commit hook](https://lets-unify.ai/ivy/deep_dive/formatting.html#pre-commit-hook) as well to make sure your commits are linted properly.
> 
> Thanks again! 😄

> I added the pre commit hook to the project so that should work, and I added a test, I was very confused on how to do it so it might be wrong but I added it. There was an error on the file before I added anything already though.

Nasser-Mohammed commented:
I followed the format of the test for regular dropout and wrote the following below in this directory: 
can you tell me if they look right: ivy_tests/test_ivy/test_frontends/test_torch/test_dropout_functions.py

<img width="660" alt="Screenshot 2023-02-24 at 1 04 44 PM" src="https://user-images.githubusercontent.com/81037208/221269505-8675fe6f-6fd7-454f-87d3-e445da4e190e.png">

<img width="715" alt="Screenshot 2023-02-24 at 1 04 58 PM" src="https://user-images.githubusercontent.com/81037208/221269523-b9dc4683-9974-4d48-a166-fd62c5d6a742.png">
